src/main/scala/com/monsanto/arch/cloudformation/model/resource/Config.scala

```diff
@@ -0,0 +1,235 @@
+                   SourceIdentifier:     Option[Token[ResourceRef[_]]] = None
+                 )
+object Source extends DefaultJsonProtocol {
+  implicit val format = jsonFormat3(Source.apply)
+}
+
+case class `AWS::Config::ConfigRule`(
+                                      name:                       String,
+                                      ConfigRuleName:             String,
+                                      Description:                String,
+                                      InputParameter:             Map[String, String],
+                                      MaximumExecutionFrequency:  ExecutionFrequency,
+                                      Scope:                      Scope,
+                                      Source:                     Source,
+                                      override val Condition:  Option[ConditionRef] = None
+                                    ) extends Resource[`AWS::Config::ConfigRule`] {
+  override def when(newCondition: Option[ConditionRef]): `AWS::Config::ConfigRule` = copy(Condition = newCondition)
+}
+object `AWS::Config::ConfigRule` extends DefaultJsonProtocol {
+  implicit val format: JsonFormat[`AWS::Config::ConfigRule`] = jsonFormat8(`AWS::Config::ConfigRule`.apply)
+}
+
+case class RecordingGroup(
+                           AllSupported:                   Option[Boolean] = None,
+                           IncludeGlobalResourceTypes:     Option[Boolean] = None,
+                           ResourceTypes:                  Option[List[ResourceType]] = None
+                         )
+object RecordingGroup extends DefaultJsonProtocol {
+  implicit val format = jsonFormat3(RecordingGroup.apply)
+}
+
+case class `AWS::Config::ConfigurationRecorder`(
+  name:               String,
+  Name:               Option[String],
+  RecordingGroup:     Option[RecordingGroup],
+  RoleARN:            Token[ResourceRef[`AWS::IAM::Role`]],
+  override val Condition:  Option[ConditionRef] = None
+) extends Resource[`AWS::Config::ConfigurationRecorder`] {
+  override def when(newCondition: Option[ConditionRef]): `AWS::Config::ConfigurationRecorder` = copy(Condition = newCondition)
+}
+object `AWS::Config::ConfigurationRecorder` extends DefaultJsonProtocol {
+  implicit val format: JsonFormat[`AWS::Config::ConfigurationRecorder`] = jsonFormat5(`AWS::Config::ConfigurationRecorder`.apply)
+}
+
+case class ConfigSnapshotDeliveryProperties(
+  DeliveryFrequency: Option[ExecutionFrequency]
+)
+object ConfigSnapshotDeliveryProperties extends DefaultJsonProtocol {
+  implicit val format = jsonFormat1(ConfigSnapshotDeliveryProperties.apply)
+}
+
+case class `AWS::Config::DeliveryChannel`(
+  name:                                 String,
+  ConfigSnapshotDeliveryProperties:     Option[ConfigSnapshotDeliveryProperties],
+  Name:                                 Option[String],
+  S3BucketName:                         Option[Token[ResourceRef[`AWS::S3::Bucket`]]],
```

Comment:
Correct, I figured we would address this all together. 

To be honest I don't think we will ever really want to make either `AWS::Config::ConfigurationRecorder` or `AWS::Config::DeliveryChannel` in practice. AWS restricts you to only having 1 per account which are generally configured through the UI.